src/components/Navbar/Account.tsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react'
+import { useRouter } from 'next/router'
+
+interface NavbarAccount {
+  isUserExist?: boolean
```

Comment:
How about `isUserSignedIn`? IMO, **exist** is more meaningful with checking null of something.